x-pack/plugins/security_solution/cypress/e2e/exceptions/shared_exception_lists_management/shared_exception_list_page/import_lists.cy.ts

```diff
@@ -0,0 +1,123 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { TOASTER, TOASTER_BODY } from '../../../../screens/alerts_detection_rules';
+import {
+  IMPORT_SHARED_EXCEPTION_LISTS_CLOSE_BTN,
+  EXCEPTIONS_TABLE_SHOWING_LISTS,
+  IMPORT_SHARED_EXCEPTION_LISTS_OVERWRITE_EXISTING_CHECKBOX,
+  IMPORT_SHARED_EXCEPTION_LISTS_CONFIRM_BTN,
+  IMPORT_SHARED_EXCEPTION_LISTS_OVERWRITE_CREATE_NEW_CHECKBOX,
+} from '../../../../screens/exceptions';
+import { closeErrorToast } from '../../../../tasks/alerts_detection_rules';
+import {
+  waitForExceptionsTableToBeLoaded,
+  importExceptionLists,
+} from '../../../../tasks/exceptions_table';
+import { visitWithoutDateRange } from '../../../../tasks/login';
+import { EXCEPTIONS_URL } from '../../../../urls/navigation';
+import { esArchiverResetKibana } from '../../../../tasks/es_archiver';
+
+describe('Import Lists', () => {
+  const LIST_TO_IMPORT_FILENAME = 'cypress/fixtures/7_16_exception_list.ndjson';
+  before(() => {
+    esArchiverResetKibana();
+  });
+  beforeEach(() => {
+    visitWithoutDateRange(EXCEPTIONS_URL);
+    waitForExceptionsTableToBeLoaded();
+  });
+  it('Should import exception list successfully if the list does not exist', () => {
+    cy.intercept(/(\/api\/exception_lists\/_import)/).as('import');
```

Comment:
Can we extract this and below line into the `beforeEach` hook?

Reply:
please check [31e49fd](https://github.com/elastic/kibana/pull/152757/commits/31e49fd99fc39a2bbf48cb06618638839b7a0bf5)